packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -176,12 +179,13 @@
 
   &::before,
   &::after {
+    @include mdc-text-field-corner-radius-all(50%);
```

Comment:
don't use the mixin for ::before and ::after elements. Then `mdc-text-field-corner-radius` mixin only applies to the `mdc-textfield` element...not it's before and after pseudo elements.

just use border-radius: 50%

Reply:
Protip: these styles will no longer exist once I update things to use new ripple mixins.

But yeah, I don't think this should be needed at all, because it should already be clipped by the parent element?